flutter/android/src/main/kotlin/com/coinbase/flutter/wallet_sdk/CoinbaseWalletSdkFlutterPlugin.kt

```diff
@@ -105,13 +107,33 @@ class CoinbaseWalletSdkFlutterPlugin : FlutterPlugin, MethodCallHandler,
     }
 
     private fun ownPublicKey(@NonNull result: Result) {
-        result.success(coinbase.ownPublicKey)
+        try {
+            val bytes = coinbase.ownPublicKey.encoded;
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                result.success(bytes?.toHexString())
+            } else {
+                result.success("")
+            }
+        } catch (e: Throwable) {
+            result.success("")
```

Comment:
Can we return an error here instead of an empty string? 
```suggestion
            result.error("ownPublicKey", e.message, null)
```